admin/server-timing.php

```diff
@@ -43,6 +43,47 @@ function perflab_add_server_timing_page() {
  * @since n.e.x.t
  */
 function perflab_load_server_timing_page() {
+	add_settings_section(
+		'output-buffering',
+		__( 'Output Buffering', 'performance-lab' ),
+		'perflab_render_server_timing_page_output_buffer_checkbox',
+		PERFLAB_SERVER_TIMING_SCREEN
+	);
+
+	add_action(
+		'admin_print_styles',
+		static function () {
+			?>
+			<style>
+				.wrap p {
+					max-width: 800px;
+				}
+
+				/* The following styles are copied from core where they normally include `.form-table td` in the selectors. */
+				.output-buffering-field {
+					margin-bottom: 9px;
+					padding-bottom: 15px;
+					line-height: 1.3;
+					vertical-align: middle;
+					font-size: 14px;
+				}
+				.output-buffering-field label,
+				.output-buffering-field p {
+					line-height: 1.4;
+				}
+				.output-buffering-field label {
+					margin: 0.35em 0 0.5em !important;
```

Comment:
```suggestion
					margin: 0.35em 0 0.5em;
```
Important is not required here as it apply cleanly.

Reply:
This is copied from core. Please make the change as you see fit.